Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -73,4 +98,18 @@ void LayoutAnimationsManager::startLayoutAnimation(
       config->getJSValue(rt));
 }
 
+int LayoutAnimationsManager::findTheOtherForSharedTransition(int tag) {
+  int theOtherTag = -1;
+  for (auto const &[key, group] : sharedTransitionGroups_) {
+    auto position = std::find(group.begin(), group.end(), tag);
+    if (position != group.end()) {
+      int index = position - group.begin();
+      if (index > 0) {
+        return group[index - 1]; // get one before
+      }
+    }
```

Comment:
Nit:
```suggestion
    if (position != group.end() && position != group.begin()) {
      return *std::prev(position);
    }
```
https://en.cppreference.com/w/cpp/iterator/prev
I think it might be more readable this way, It's fine if you prefer the index approach though.